stdlib/http/src/main/ballerina/src/http/service_endpoint.bal

```diff
@@ -77,19 +77,10 @@ public type Listener object {
     # Gets invoked during module initialization to initialize the listener.
     #
     # + port - Listening port of the HTTP service listener
-    # + c - Configurations for HTTP service listener
-    public function __init(int port, public ListenerConfiguration? config = ()) {
```

Comment:
We need the docs for config here right?

Let's add
```
# + config - Configurations for HTTP service listener
```

Reply:
Done